### PR TITLE
docs(aio): fixed grammar and typos

### PR DESCRIPTION
**Pull request by sibiraj-s**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)
grammar mistakes and typos

**What is the new behavior?**
fix grammar and typos

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

### CODE REVIEW

**Comment by Sibiraj-S** [comment]
I signed it!

**Comment by Sibiraj-S** [comment]
@gkalpak . Can this be merged? as it is getting outdated from the current one.

**Comment by gkalpak** [comment]
cc @wardbell

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
             "url": "https://github.com/NativeScript/nativescript-angular"
           },
           "ab5": {
-            "desc": "An Universal Windows App (uwp) powered by Angular 2",
+            "desc": "A Universal Windows App (uwp) powered by Angular 2",
```
Comment (wardbell) [inline aio/content/marketing/resources.json]: The merge conflict is here. Please replace "Angular 2" with "Angular"

**Review by wardbell — CHANGES_REQUESTED** [review]
LGTM except as noted and you have to resolve the merge conflicts. I would have fixed it myself but I can't update your branch.

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (Sibiraj-S) [inline aio/content/marketing/resources.json]: since I made the change. a week ago. I deleted my branch. Is there anything else can be done

Comment (Sibiraj-S) [inline aio/content/marketing/resources.json]: ?

Comment (Sibiraj-S) [inline aio/content/marketing/resources.json]: what to do now @wardbell

Comment (Sibiraj-S) [inline aio/content/marketing/resources.json]: can this get cleaned up while merging?

**Comment by Sibiraj-S** [comment]
@wardbell opened a new PR #17782 for the same since I deleted the branch for this PR.  Both have same changes but up to date with master

**Comment by gkalpak** [comment]
Closing in favor of #17782.

**Pull request closed without merging**